libr/anal/p/anal_arc.c

```diff
@@ -420,26 +419,28 @@ static int arcompact_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *data, in
 		return 0;
 	}
 
+	op->type = R_ANAL_OP_TYPE_UNK;
```

Comment:
yeah - as a newcomer, I just assumed that the op struct was initialised before being handed to the anal plugin